### DISCUSSION TITLE
Edit on github link doesnt work on docs.ulauncher.io

### DISCUSSION THREAD

**Discussion by wre9-tesh** [Ulauncher/Ulauncher#1157] [General]
I want to fix this issue .I found that this can be fix by just changing dev to v6 in the url for the link 

ex -https://github.com/Ulauncher/Ulauncher/blob/dev/docs/index.rst  to https://github.com/Ulauncher/Ulauncher/blob/v6/docs/index.rst

I am new to this project  and open-source in general .what should i do to understand the codebase to fix this?

**Comment by friday** [comment]
The problem is with the readthedocs config (docs.ulauncher.io is a readthedocs page). I don't know how that works and the author who set it up doesn't have time to contribute.

Feel free to make a PR for the actual pages, but they won't be updated publicly until this is fixed by me or the author.
